# Generated by Django 4.0.4 on 2022-11-04 16:15

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('exceptions', '0008_aftexceptioncode_operator_msg'),
    ]

    operations = [
        migrations.CreateModel(
            name='AFTExceptionCodeManifest',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('created', models.DateTimeField(auto_now_add=True)),
                ('lastModified', models.DateTimeField(auto_now=True)),
                ('version', models.CharField(max_length=31)),
                ('manifest', models.JSONField()),
                ('creator', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='%(app_label)s_%(class)s_creator_related', to=settings.AUTH_USER_MODEL)),
                ('modifiedBy', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='%(app_label)s_%(class)s_modifiedby_related', to=settings.AUTH_USER_MODEL)),
            ],
            options={
                'abstract': False,
            },
        ),
        migrations.AddField(
            model_name='aftexceptioncode',
            name='manifest',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to='exceptions.aftexceptioncodemanifest'),
        ),
    ]
